### DISCUSSION TITLE
How can I save something to memory

### DISCUSSION THREAD

**Discussion by Somecoolname444333** [WongSaang/chatgpt-ui#171] [Q&A] [answered]
I don't understand how I can add something to the persistent memory. If I tell in the chat so remember something or save it to his internal memory he will forget it immediately. Also is there a way to train it with some custom information (ex. PDF library?)

**Comment by WongSaang** [comment, marked as answer]
Hello, if you want it to remember your conversation content, please turn off the frugal mode. However, memory is still limited depending on the token limit of the model used. This project is only a client for chatgpt, so it cannot be trained.
